### PR TITLE
refactor: ignore markdown on metadata

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

note article observer and markdown parsing is based on https://github.com/ArdentHQ/dashbrd/pull/272

Closes https://app.clickup.com/t/862kh4tf2

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: app/Observers/ArticleObserver.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        $this->forgeMetaDescriptionCache($article);
+    }
+
+    public function updated(Article $article): void
+    {
+        $this->forgeMetaDescriptionCache($article);
+    }
+
+    private function forgeMetaDescriptionCache(Article $article): void
```
Comment (shahin-hq) [inline app/Observers/ArticleObserver.php:22]: `forgeMetaDescriptionCache` -> `forgetMetaDescriptionCache` it should be

**Comment by ItsANameToo** [comment]
draft until #272 is merged

**Comment by ItsANameToo** [comment]
conflicts now that #272 is merged @alfonsobries

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Review by ItsANameToo — APPROVED** [review]

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**